hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java

```diff
@@ -29,6 +34,25 @@
 public class LocalMapStatsTest extends HazelcastTestSupport {
 
     @Test
+    public void testMapStatsAreNotLeaking() throws Exception {
```

Comment:
It doesn't imply that there no leakage of statistics objects. It could happen that statistics are reset when map is destroyed and the test will pass even they are leaking.

To me both ways is ugly/hacky.